draft-ietf-core-new-block.xml

```diff
@@ -301,27 +302,37 @@
 
       <section anchor="scope" title="Applicability Scope">
         <t>The block-wise transfer specified in <xref target="RFC7959"></xref>
-        covers the general case, but falls short in situations where packet
-        loss is highly asymmetrical. The mechanism specified in this document
+        covers the general case using Confirmable messages, but falls short in
+        situations where packet
+        loss is highly asymmetrical or there is no need for an
+        acknowledgement. In other words, there is a need for Non-confirmable
+        support.</t>
+
+        <t>The mechanism specified in this document
         provides roughly similar features to the Block1/Block2 Options. It
         provides additional properties that are tailored towards the intended
         use case of Non-Confirmable transmission. Concretely, this mechanism
         primarily targets applications such as DDoS Open Threat Signaling
-        (DOTS) that cannot use CON responses to handle potential packet loss
-        and that support application-specific mechanisms to assess whether the
+        (DOTS) that cannot use CON requests/responses because of potential
+        packet loss
+        and that ideally support application-specific mechanisms to assess
+        whether the
         remote peer is not overloaded and thus is able to process the messages
         sent by a CoAP endpoint (e.g., DOTS heartbeats in Section 4.7 of <xref
-        target="RFC8782"></xref>).</t>
+        target="RFC8782"></xref>). Other use cases are when an application
+        needs to send data but have no need for an acknowledgement of
```

Comment:
to avoid two "needs" in the same sentence:

```suggestion
        have to send data but have no need for an acknowledgement of
```

Reply:
Updated, but wording changed as well.